packages/nimble-components/src/patterns/dropdown/styles.ts

```diff
@@ -242,12 +243,12 @@ export const styles = css`
 
     .scrollable-region {
         overflow-y: auto;
+        scroll-snap-type: y mandatory;
```

Comment:
Tried the storybook for Chrome on Windows and it ends up feeling kinda clunky. The scroll on open is too slow and feels like it gets in the way. It also looks like Chrome on Windows using the mousewheel to scroll does not have kinetic scrolling or something. So it always stops then shifts again and feels pretty uncomfortable.

I tried to record a gif but it's not showing enough frames to see the issue. The shift on Option 47 is what I see on each option as I'm scrolling:

![scrollsnapwinchrome](https://github.com/user-attachments/assets/7a951d2e-204c-4fc4-bde8-7e42d9f2c8b2)

Reply:
I don't see the same with a Windows 11 VM + mouse scroll wheel, but I've pulled the change out of this PR, and we can evaluate if these APIs are well enough supported to use in another PR.